OPCB/functions/missions/Destroy.sqf

```diff
@@ -4,22 +4,40 @@ _taskcomp = selectRandom ["warhead1","warhead2","destroy2"];
 _guardgroup = createGroup [east,true];
 
 _current_task = _base getPos[random 600,random 360];
-[_current_tasknumber ,west,["Last night an american Tank was stolen. You are tasked with destroying it as quickly as possible, before media notice","Destroy"], _current_task,"ASSIGNED",10,true,true,"Destroy",true] call BIS_fnc_setTask;
+[_current_tasknumber ,west,
+["Last night an american Tank was stolen. You are tasked with destroying it as quickly as possible, before media notice","Destroy"],
+ _current_task,"ASSIGNED",10,true,true,"Destroy",true] call BIS_fnc_setTask;
+
 _guard = _guardgroup createUnit [OPCB_unitTypes_inf_ins_commander, _base, [], 2, "NONE"];
 _guardpos = getpos _guard;
 _spawncomps = [_guard] call CHAB_fnc_roadblock_rus;
+
 _comp = [_taskcomp,_guardpos, [0,0,0], random 360, true, true ] call LARs_fnc_spawnComp;
 [_guard,10,2,1] call CHAB_fnc_spawn_nat;
+
 _thetarget = createVehicle ["rhsusf_m1a1aimd_usarmy", _guardpos, [], 1, "NONE"];
 _thetarget lock true;
-waitUntil { sleep 10; !canMove _thetarget};
+
+waitUntil {
+  sleep 10;
+  !alive _thetarget || {damage _thetarget > 0.8}
```

Comment:
Yep it's definitely not ideal. You're not overwriting the mission's CBA settings with your own "server" ones are you? You should make sure all server override boxes are unchecked for the addon settings. Anyway, it's strange that it worked for you because that was a known and common issue. 

We could implement a crude damage event handler here that checks for particular explosives being used and if that's the case kills the tank by command. Actually, this also brings to mind the problem with being able to finish any kind of "destroy" task from a distance, which I also had in mind some time ago... For balancing reasons, maybe we should start thinking of enforcing by script a "hand-planted explosive charge only" policy to successfully complete those missions....

Reply:
I haven't even played on the server yet, I don't know what CBA settings are :D never touched them. 
I'll put back the canMove command for now in this case, I believe you. 

But in the future we should refactor the destroy tasks for something more ideal . 

I have had a defuse bomb mission back in the day where you had to cut a wire or die trying. The wire was either a hidden clue in one of the buildings or you had to have someone at base to check in with jeff about explosive types. Basically you couldn't just cut the wires randomly. (It was all implemented and tested, but turned out to be too 'complex' and needed too much 'teamwork' therefore it was discaded. Can we bring this idea back ? I really loved that task.

Reply:
CBA settings are the "addon settings" in the editor and in-game pause menu. When testing in the editor, you still have a "server settings" tab that is unique to your environment and can override the addon settings set by the mission if you have some checkboxes checked. That's what I was asking about. I'm not actually talking about our server, where it's already set to not override mission settings.

That task idea sounds awesome! Yes, we should bring that back! >:D